asreview/webapp/src/PreReviewComponents/ProjectInfo.js

```diff
@@ -96,26 +110,17 @@ const ProjectInfo = (props) => {
     evt.preventDefault();
 
     var bodyFormData = new FormData();
+    bodyFormData.set("mode", info.mode);
     bodyFormData.set("name", info.name);
     bodyFormData.set("authors", info.authors);
     bodyFormData.set("description", info.description);
 
-    (props.edit
-      ? ProjectAPI.info(props.project_id, true, bodyFormData)
-      : ProjectAPI.init(bodyFormData)
-    )
```

Comment:
All lines involving the removal of `props.edit` will affect the functionality of editing the project info. Currently, this initiating a new project function and editing the project info function share the same dialog. For now, I would propose to keep `props.edit` as an indicator of whether this dialog is used as a function to initiate or edit a project. And when the dialog plays the function of editing the project info, no selection menu for the mode is displayed.
In the future, I expect these two functions to be separated. The function to initiate a project should be integrated into the setup stage.

Reply:
Thanks, great remark. I will try to resolve these conflicts and make the edit function work again.

Reply:
I think I resolved this.